modules/core/src/main/scala/lucuma/odb/api/model/targetModel/SourceProfileModel.scala

```diff
@@ -233,47 +234,77 @@ object SourceProfileModel {
         a.measure
       )}
 
+    def measure[T]: Lens[BandBrightnessPair[T], Measure[BrightnessValue] Of Brightness[T]] =
```

Comment:
Should this be in core?

Reply:
I resurrected the `BandBrightnessPair` which was in an earlier incarnation of the source profile model because it was useful to me.  Since the class itself only exists in the API now, the `Lens` has to go there too.